buildDockerImages.pl

```diff
@@ -195,6 +195,10 @@ sub setupForBuild {
 
 sub cleanupAfterBuild {
 	my ($fileout) = @_;
+	#cleaning extraneous files from previous runs
+	runAndLog($fileout, "docker rmi -f \$(docker images -q)");
+	runAndLog($fileout, "docker container prune -f");
```

Comment:
The first line removes all docker images including non weathervane related ones?
The second line removes all containers including non weathervane related ones?
I don't think every user wants to do either of these by default and maybe not even optionally as part of the script -- they can always been done manually if needed.
Is there a straight forward way to prune by some filter just the weathervane related containers typically left behind?

Reply:
That's a good point, the same thought crossed my mind. I'll see if there's a way to filter which gets deleted and which doesn't.